src/utils.ts

```diff
@@ -27,3 +29,15 @@ export async function randomArray(size: number): Promise<string> {
         });
     });
 }
+
+type JsonValue = null | string | number;
+type JsonArray = Array<JsonValue | JsonObject | JsonArray>;
+interface JsonObject {
+    [key: string]: JsonObject | JsonArray | JsonValue;
+}
+type Json = JsonArray | JsonObject;
+
+export function loadJsonFile<T extends Json>(...paths: string[]): T {
```

Comment:
document this please. why are there several paths? how does the behaviour end up differing from `require`?